### ISSUE TITLE
"invalid" version in `package.json`

### ISSUE THREAD

**Issue by kenany** [BryceMatthes/betterdgg#83]
Can't run any `npm` commands now because the `"version"` in the `package.json` is invalid. [By definition](https://docs.npmjs.com/files/package.json#version), the `"version"` field must be valid [semver](http://semver.org/spec/v2.0.0.html), which `1.4.4.3` is not. Version number should be of the form `X.Y.Z` (gross simplification of the spec, but should be sufficient for us) so we should stick to that format.

``` bash
$ npm install 
npm WARN Invalid version: "1.4.4.3"
# nothing installed

$ npm run
npm ERR! Linux 4.8.13-1-ARCH
npm ERR! argv "/home/kenan/bin/node" "/home/kenan/.npm-packages/bin/npm" "run"
npm ERR! node v7.4.0
npm ERR! npm  v4.1.1

npm ERR! Invalid version: "1.4.4.3"
```

**Comment by downthecrop** [comment]
Got it. My bad 😬

**Comment by downthecrop** [comment]
Issue resolved in 1.4.5 ba07f9d
